pennylane/optimize/adagrad.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Adagrad optimizer"""
-from pennylane.numpy import sqrt
+from numpy import sqrt
```

Comment:
So the optimizers are actually autograd-specific, so we may actually need to keep `pennylane.numpy` here.

Reply:
not totally sure, although I thought the motivation for all this was to make things like the optimizers not-autograd-specific right? tests are passing, so it seems fine. I think `sqrt` is only used to a non-autograd value, then subtracted from the actual variable being trained.